### PR TITLE
fixed ClassDB inconsistencies

### LINKED ISSUE #7960: Some inconsistencies in classDB's bindings
Hi,

I'm currently using classDB introspection system to generate bindings for godot-python.
I've found some troubles in the bindings:

- audio_server.cpp: `AudioServer.set_state` and `AudioServer.generate_state` are mark to return a `AudioServerState`, but there is no reference of this type anywhere
- core_bind.cpp: `_Marshalls::base64_to_raw` is marked to return a `RawArray` which is an out-of-date alias to `PoolByteArray` (i.e. `PoolVector<uint8_t>`)
- visual_script: `VisualScriptPropertySet` properties' names start with `property/...`, this is a pretty strange name given slash is not a valid variable character for most languages (including C++ and GDscript) so doing `VisualScriptPropertySet.property/set_mode` is impossible...
- control.cpp: `Control.get_minimum_size` is binded two times (has a virtual and a regular method)
- packet_peer.cpp: `PacketPeer.put_packet` is declared has returning an `Error` type which is never defined within classdb

I'm ready to send a PR to solve this (seems all minor issues to me, but some for them like the `property/...` stuff is more like something I didn't catch...)

### PR DESCRIPTION
**Pull request by karroffel**
partially fixes #7960 (The property problem isn't solved)

### CODE REVIEW

File: scene/gui/control.cpp [hunk 1]
```diff
@@ -2419,7 +2419,7 @@ void Control::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("_font_changed"), &Control::_font_changed);
 
 	BIND_VMETHOD(MethodInfo("_gui_input", PropertyInfo(Variant::INPUT_EVENT, "event")));
-	BIND_VMETHOD(MethodInfo(Variant::VECTOR2, "get_minimum_size"));
+	BIND_VMETHOD(MethodInfo(Variant::VECTOR2, "_get_minimum_size"));
```
Comment (bojidar-bg) [inline scene/gui/control.cpp]: That's incorrect, you forgot to call the new method..... (you should probably edit SceneStringNames too, IDK)

**Review by bojidar-bg — CHANGES_REQUESTED** [review]

File: scene/gui/control.cpp (hunk 1 above)
Comment (karroffel) [inline scene/gui/control.cpp]: Call the new method, what do you mean? And why? All other virtual methods don't use SceneStringName as well

Comment (karroffel) [inline scene/gui/control.cpp]: Ohhhh I see what you mean. Yeah, will do.

Comment (karroffel) [inline scene/gui/control.cpp]: @bojidar-bg updated

**Review by bojidar-bg — APPROVED** [review]
Ok, no more comments from my side, but some other reviewer might want to have a look at it :smiley:

**Pull request merged**